src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball ballOne = Lottery.getRandomBall();
+        Ball ballTwo = Lottery.getRandomBall();
+        Ball ballThree = Lottery.getRandomBall();
+        System.out.println(ballOne);
+        System.out.println(ballTwo);
+        System.out.println(ballThree);
```

Comment:
Use for loop for creating several objects of the same class https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class